lib-js/test-case/compare.ts

```diff
@@ -83,7 +83,7 @@ export function compareResults(
     return failures.MissingOutput();
   } else if (expected.isSuccess) {
     if (!actual.isSuccess) {
-      return failures.UnexpectedError();
+      return failures.UnexpectedError(actual.error);
```

Comment:
BTW I should mention for context - if you look at the bug report the error is very cryptic and not easy to read at all, by adding this I found the error to be more useful